README.md

```diff
@@ -101,6 +101,12 @@ To see full configuration types see [template.lua](./lua/leetcode/config/templat
     ---@type boolean
     logging = true,
 
+    ---@type boolean
+    toggle_console_on_run = false,
+
+    ---@type boolean
+    toggle_console_on_submit = false,
+
     console = {
```

Comment:
This would a better place for this. I know the readme states that this is for appearance only so i will have to change that later 
```suggestion
    console = {
        ---@type boolean
        open_on_runcode = false,
```